### DISCUSSION TITLE
using unique processed site dirs with addons?

### DISCUSSION THREAD

**Discussion by leonstafford** [WP2Static/wp2static-addon-s3#43] [Q&A]
@john-shaffer is this by design or likely an auto-code cleanup issue I introduced?

`$s3_path = \WP2Static\SiteInfo::getPath( 'uploads' ) . 'wp2static-processed-site.s3';`

**Comment by john-shaffer** [comment]
It looks like s3_path and s3_url are unused and can be safely removed. I don't think it was auto-code, just some stray code that was never removed.

**Reply by leonstafford** [reply]
Ah, awesome, cheers!
